en/editor/publish/index.md

```diff
@@ -1,33 +1,43 @@
-# Publish Workflow
+# Cross-platform Game Publish
 
-The **Build** panel allows you to publish games to multiple platforms, including **Web**, **iOS**, **Android**, **Windows**, **Mac**, and various **mini-game** platforms.
+Cocos Creator supports all major platforms in depth, games can be quickly published to Web, iOS, Android, Windows, Mac, and various mini-game platforms, truly realizing one development and running on all platforms.
 
-## Index
+## Preparation Work
+
+The game project can be published when it is developed normally and the preview results meet the requirements. The preparatory work that need to be done before publishing includes:
 
 - [About the Build Panel](build-panel.md)
-- [General Build Options](build-options.md)
-- [Publish from the Command Line](publish-in-command-line.md)
+- [Build Options](build-options.md)
+
+## Publishing Platform
+
+Currently, Cocos Creator supports publishing platforms as follows:
```

Comment:
Currently, Cocos Creator supports publishing platforms as follows:
-->
Currently, Cocos Creator supports publishing to the following platforms: